#objects_info with vel and dispersion of vel

point3d[] pos
float64[] r
point3d[] vel
point3d[] dsp
cluster_point[] pt
